nss-day-cohort-60/django-taters-api
# login and register updated with encrypted token

Closes #32
Closes #33
Closes #34 


# Description of Proposed Changes

Added auth.py, login_user, register_user
---


# Steps to Test

1. Fetch the jes-login-refactor CLIENT branch and switch to it, NPM Start
2. Fetch the jes-login-register-logout branch and switch to it, start debugger
3. Click Register on the landing page, attempt to register a new user, verify user has been added to your local DB
4. Logout, verify user is out of local storage
5. Click Login, verify you can log in with your recently created User
